airflow/lineage/datasets.py

```diff
@@ -18,7 +18,10 @@
 # under the License.
 import six
 
-from jinja2 import Environment
+try:
+    from jinja2.nativetypes import NativeEnvironment as Environment
```

Comment:
We should just update the minimum version of Jinja we require instead.

Reply:
> We should just update the minimum version of Jinja we require instead.

Updating the minimum version of Jinja cannot resolve the #4770 (list as templated field issue). Referring to your suggestion in that PR, we can either import jinja2.NativeEnvironment or convert specific templated fields into list type in those operators.

Reply:
Sorry, I  mean update jinja version to one that always has NativeEnvironment - so we don't have to have the except ImportError.

Reply:
You are right! Thanks for the suggestion!